src/schema/types/product.graphql

```diff
@@ -5,6 +5,7 @@ type Product {
   description: String
   price: Float!
   marketPrice: Float!
+  productPhotos: [ProductPhoto]!
```

Comment:
Do we need to call this argument productPhotos? Why don't we call it just photos?

Reply:
Yup, you're right